### ISSUE TITLE
ones is not defined

### ISSUE THREAD

**Issue by paritosh-gupta** [bjbschmitt/AMFM_decompy#3]
Issue occurs on line https://github.com/bjbschmitt/AMFM_decompy/blob/da1751e87120b795c058b840df9e95949085c8f7/amfm_decompy/pYAAPT.py#L526

I think it needs to be np.ones ?
I got this as an error while going through an audio file.

**Comment by paritosh-gupta** [comment]
I changed it on my local machine and then got this error.
```
/usr/local/lib/python2.7/dist-packages/AMFM_decompy-1.0.6-py2.7.egg/amfm_decompy/pYAAPT.pyc in spec_track(signal, pitch, parameters)
    524     else:
    525         if num_voiced_cand > 0:
--> 526             voiced_pitch = (np.ones(1, num_voiced_cand))*150.0
    527         else:
    528             voiced_pitch = [150.0]

/home/paritosh/Documents/Oratio/oratio-scripts/f0-python/numpy/core/numeric.pyc in ones(shape, dtype, order)
    190 
    191     """
--> 192     a = empty(shape, dtype, order)
    193     multiarray.copyto(a, 1, casting='unsafe')
    194     return a
```

**Comment by bjbschmitt** [comment]
Hi, thanks for your report!

Yes, it has to be np.ones indeed. The second error is probably caused because the input for np.ones has to be a python tuple. Try changing line 526 to 

voiced_pitch = (np.ones((1, num_voiced_cand)))*150.0

These two extra parentheses make the code uglier, but it may solve the problem. 

Anyway, it seems that during my tests the if statement from line 525 was never accessed, so these two errors passed unnoticed. Could you please send me your problematic audio file, if possible? I want recreate the issue here just to check if these modifications are enough.

**Comment by bjbschmitt** [comment]
Ok, since that Paritosh didn't help us by sending a feedback after a week, I'll consider that this issue is fixed. The proposed alterations have been included to the code and the new version 1.0.6.1 has been updated here in github and in pypi.

I couldn't recreate the issue in my PC to check with 100% certain that the modifications are enough. But since that it's a simple problem, I believe that they will do the work.
